doc/source/how-to/documenting.rst

```diff
@@ -506,49 +505,266 @@ Your next step will be to upload the documentation artifact. Assuming your
-      uses: JamesIves/github-pages-deploy-action@4.3.0
-      with:
-        branch: gh-pages
-        folder: doc/build/html
-        clean: true
+The following job step shows the logic for deploying. If you wish to deploy to
+another repository, make sure to uncomment the ``repository-name`` line and
+declare the name of your documentation repository:
+
+.. tabs::
+
+    .. group-tab:: Using ``tox``
+
+        .. code-block:: yaml
+        
+            - name: Deploy
+              if: github.event_name == 'push' && contains(github.ref, 'refs/tags')
+              uses: JamesIves/github-pages-deploy-action@4.3.0
+              with:
+                token: ${{ secrets.GITHUB_TOKEN }}
+                # repository-name: pyansys/repository-name
+                branch: gh-pages
+                folder: .tox/doc_out
+                clean: true
+
+    .. group-tab:: Without Using ``tox``
+
+        .. code-block:: yaml
+
+            - name: Deploy
+              if: github.event_name == 'push' && contains(github.ref, 'refs/tags')
+              uses: JamesIves/github-pages-deploy-action@4.3.0
+              with:
+                token: ${{ secrets.GITHUB_TOKEN }}
+                # repository-name: pyansys/repository-name
+                branch: gh-pages
+                folder: doc/_build/html
+                clean: true
+
+
+Notice that for previous job steps, a ``GITHUB_TOKEN`` is required. GitHub
+automatically generates the token ``GITHUB_TOKEN`` which you can use as
+``token`` for deploying documentation to the same repo. However, if you are
+planning to deploy to another repo this token does not have the permissions to
+do it.
+
+In that case, there are two options for documentation deployment, using a bot
+and using a personal access token (PAT). Depending on your profile (how many
+organizations you work in, the rights in the different repositories, etc.), using
+a PAT can be potentially dangerous because PAT are not restricted to defined
+repositories, rather have general permissions. This means that a PAT with
+`repository-write` permission can write in any repo in any organization that the
+PAT creator can access.
+
+Therefore, the recommended approach is to **use a Bot**. However, you are free
+to use a PAT if you feel it fits your needs better.
+
+
+Deploying by Using a Bot
+""""""""""""""""""""""""
+To deploy documentation to repo different than the one where the documentation
+is generated, you need permissions to access (*read/write*) this second repo.
+These permissions can be handled using a specifically created bot. In PyAnsys
+organization, there is `PyAnsys Bot`_ which has read and write permission across
+some repositories and can be used for this purpose.
+
+.. admonition:: Organization approval to use PyAnsys bot
+
+    You need internal approval to use PyAnsys bot as your repo needs to be added to
+    its list of repositories.
+    Please email `PyAnsys Support <pyansys.support@ansys.com>`_.
```

Comment:
```suggestion
    You need internal approval to use PyAnsys bot because your repository must be added to
    its list of repositories. For more information, email `PyAnsys Support <pyansys.support@ansys.com>`_.
```